modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -812,7 +812,8 @@ m|+++0+++
 |Description
 a|Names of the databases allowed on this server; all others are denied. Empty means all are allowed. This configuration can be overridden when enabling the server or altered at runtime without changing this setting. Exclusive with `server.initial_denied_databases`.
 |Valid values
-a|A comma-separated set where each element is a valid database name containing only alphabetic characters, numbers, dots, and dashes with a length between 3 and 63 characters, starting with an alphabetic character but not with the name `system`.
+a|A comma-separated set where each element is a valid database name.
+Database naming rules are described in xref:database-administration/standard-databases/naming-databases.adoc[Naming rules for databases]
```

Comment:
There is a build error regarding this file, I guess something with this line?

Reply:
Oh, it seems like they validate the config descriptions against the database, so I can't do this: 
```
   == Docs ==
    A valid database name. Database naming rules are described in Naming rules for databases
    == Neo4j ==
    A valid database name containing only alphabetic characters, numbers, dots, and dashes with a length between 3 and 63 characters, starting with an alphabetic character but not with the name system.
```
I guess I need to go back into the monorepo and update them all there :face_exhaling:

Reply:
Actually that doesn't make much sense to have "from version ..." in the database core repo, since that always knows what version it is. Maybe I just need to add the digits there and leave it at that.

Reply:
We can add labels in the docs description as labels are ignored by the test if by "from version .." you mean when this change is introduced.

Reply:
https://github.com/neo-technology/neo4j/pull/28262